src/components/Resources/ResourcePage.js

```diff
@@ -22,6 +30,8 @@ function ResourcePage({ matchProps }) {
       });
   }, [matchProps.match.params.id]);
 
+  const classes = useStyles();
+
   return (
     <>
```

Comment:
I don't think this needs to be rendered as a fragment here.

Reply:
++ (@bengineerdavis -- docs on fragments at https://reactjs.org/docs/fragments.html btw)

Reply:
Thanks for the help! Fixed this with e3cb334.